### ISSUE TITLE
Auto-crit not calculating damage for default scout dmg

### ISSUE THREAD

**Issue by SylveonBottle** [Fedoraware/Fedoraware#835]
Doesn't all of scout primaries deal less than base damage?
I'm noticing auto-crit getting crit kills in the kill feed (when it shouldn't) with scout's melees
fyi base scout does 35 dmg,

https://github.com/Fedoraware/Fedoraware/commit/ad5dea408996ba93acaf429e39fa02af6b9ab939

**Comment by xxmyn** [comment]
r u sure that's the right commit?

**Comment by SylveonBottle** [comment]
> r u sure that's the right commit?
no I am not.
updated

**Comment by xxmyn** [comment]
I'm a bit confused but it seems like it's doing what it's meant to, if it can't kill the player with a non crit, it will crit, the rounding seems fine except for the hothand because it attacks twice?

**Comment by SylveonBottle** [comment]
It's been really long time since I made this, so I'm not quite sure if I remember why I created it.

If I recall, I believe it's because base damage is 65 or something. However when you swing with the scout that's, autocrit incorrectly assumes that the default bat deals 65 damage per swing. 

All of Scout's weapons deal less damage by default. This wasn't accounted for in the base code, so any of Scout's weapon that didn't specifically have another speed attribute would incorrectly not use autocrit.

The problem may have been fixed since then?
